decimal64_test.go

```diff
@@ -1,10 +1,10 @@
 package decimal
 
 import (
+	//"fmt"
+	"github.com/stretchr/testify/require"
```

Comment:
Original placement was better.

Reply:
`goimports` and `gofmt` should ideally be executed on every save. You can probably set your editor up this way so you don't have to worry about it.